spec/controllers/pages_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe PagesController, type: :controller do
+
+  describe 'landing page' do
+
+    context 'when it is loaded' do 
```

Comment:
`context` must never do nothing.

In this case it's doing nothing - it's a wrapper block which doesn't provide anything.

`context` blocks must always be accompanied of stuff like:

* `let` / `def`
* `before` / `after`

In this case, you can move `get 'index'` to a `before` block belonging to the `context` block. That way the `context` would do something - there would be a relationship between the text `'when it is loaded'` and an actual piece of code.